### ISSUE TITLE
Separate difficulties

### ISSUE THREAD

**Issue by stampapiede** [osuripple/ripple#124]
Lno submitted a feature request: **Separate difficulties**
```
Allow for each user to select independent difficulties on maps in multi player.
```
Despite being a smart bot, I can not automagically respond to the user if you have any feedback to give them. Thus, if you wish to tell 'em something, respond here: https://ripple.moe/index.php?p=114&id=61

**Comment by xNyo** [comment]
wut?

**Comment by marcostudios** [comment]
Chiede di poter scegliere di giocare a diverse difficoltà ma stessa mappa,
cosa non possibile da client credo

Il lun 4 apr 2016 10:06 Nyo <notifications@github.com> ha scritto:

> Closed #124 <https://github.com/osuripple/ripple/issues/124>.
>
> —
> You are receiving this because you are subscribed to this thread.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/osuripple/ripple/issues/124#event-613220061>
>
-- 

Ora uso un nuovo indirizzo email (marco.bassetto@protonmail.com), se vuoi
rispondermi per favore usa quello.
